### ISSUE TITLE
Not catching errors/warnings where message has newlines

### ISSUE THREAD

**Issue by bstoutenburgh** [maristgeek/SublimeLinter-contrib-bashate#10]
An example warning from `heredoc_ignore.sh`:

```
[W] E006: Line too long: 'FOO=<<EOF

 this is a file
  that does not obey our indenting
 or our line length -------------------------------------------------

EOF'
 - heredoc_ignore.sh : L9
```

The regex in `linter.py` does not catch this.

**Comment by bstoutenburgh** [comment]
Not a bug with this project, this was fixed in bashate itself. Not sure what version of bashate I was using that generate this problem but it looks like after 0.4.0 bashate ignores heredoc lines for all checks based on commits made Feb 8 2016.
